cookiewow-wordpress.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: Cookie Wow WordPress
  * Plugin URI: https://cookiewow.com/
- * Description:
+ * Description: An easy way to manage cookie consent on web pages.
```

Comment:
This text will be displayed at the WordPress backend, Plugins page.

<img width="1258" alt="Screen Shot 2564-02-25 at 17 42 58" src="https://user-images.githubusercontent.com/4145121/109142080-07d07f80-7791-11eb-9c90-77d45e587553.png">